contracts/test/MockERC20Permit.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.17;
-
+import "../IERC20WithPermit.sol";
```

Comment:
Doesn't OZ have an interface contract for permit tokens? Must've missed where this interface was added to the codebase

Reply:
I added it during the removal of circular deps. They do have this [draft-IERC20PermitUpgradeable.sol](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/token/ERC20/extensions/draft-IERC20PermitUpgradeable.sol) Which has the Permit extensions but not the rest of the ERC20 spec.

Reply:
Makes sense then!